templates/test/prow/prow-ci-version/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
is this not the same than what's in templates/test/prow-ci-version ?

Reply:
Accidentally checked in extra files. I will be deleting them.